JS/add.js

```diff
@@ -0,0 +1,54 @@
+const add = document.getElementById('add');
+const buttons = document.querySelectorAll('.deleteBook');
+
+const myBooks = document.getElementById('books');
+const inputsAreValid = (form) => {
+  const allInput = form.querySelectorAll('input');
+  const inputArr = [...allInput];
+  return inputArr.every((input) => input.value.trim().length > 2);
+};
+
+const addBook = (e) => {
+  e.preventDefault();
+
+  const form = document.querySelector('form');
+
+  const storedCollection = JSON.parse(localStorage.getItem('mybooks'));
+
+  const id = `${storedCollection.length + 1}`;
+  const newBook = {
+    id,
+    title: form.title.value,
+    author: form.author.value,
+  };
```

Comment:
Thank you for your observation. But do you mean the book cannot be removed until the page has been refreshed? Because the book is added to the list as seen in my console without refreshing the page

Reply:
Please reply as soon as you can